packages/inngest/src/components/connect/index.ts

```diff
@@ -171,18 +160,27 @@ class WebSocketWorkerConnection implements WorkerConnection {
     // If the previous connection was already closed, this will be a no-op.
     await this.cleanup();
 
-    // In case the previous connection disconnected, we still need to wait for all
-    // in-flight requests to complete and flush buffered messages.
-    if (this.cleanupBeforeClose) {
-      this.debug("Running cleanup before close");
-      await this.cleanupBeforeClose();
-      this.cleanupBeforeClose = undefined;
-    }
+    this._connectionId = undefined;
+    this.currentWs = undefined;
+    this.lastGatewayHeartbeatAt;
```

Comment:
Should this be reset too?
```suggestion
    this.lastGatewayHeartbeatAt = undefined;
```

Reply:
Yeah good catch, thanks!